docs/getting-started/Legacy_ESLint_Setup.mdx

```diff
@@ -84,10 +84,40 @@ ESLint will lint all TypeScript compatible files within the current folder, and
 
 ## Next Steps
 
-We provide a plethora of powerful rules that utilize the power of TypeScript's type information. [Visit _Typed Rules_ for a setup guide](./Typed_Linting.mdx).
-
 If you're having problems getting this working, please have a look at our [Troubleshooting & FAQs](../troubleshooting/FAQ.mdx).
 
+### Additional Configs
+
+We provide two shared configs that we recommend you consider enabling:
+
+- [`strict`](../users/Shared_Configurations.mdx#strict): a superset of `recommended` that includes more opinionated rules which may also catch bugs.
+- [`stylistic`](../users/Shared_Configurations.mdx#stylistic): additional rules that enforce consistent styling without significantly catching bugs or changing logic.
+
+```js title=".eslintrc.cjs"
+/* eslint-env node */
+module.exports = {
+  extends: [
+    'eslint:recommended',
+    // Remove this line
+    'plugin:@typescript-eslint/recommended',
+    // Added lines start
+    'plugin:@typescript-eslint/strict',
+    'plugin:@typescript-eslint/stylistic',
+    // Added lines end
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  root: true,
+};
+```
```

Comment:
I don't know if we should include a config block in here?
People will follow the bouncing ball and just do it without thinking... Which is a good and a bad thing.

Reply:
@bradzacher what do you mean by "include a config block"?

Reply:
Ah gotcha. Hmm. I like having it here - if they've reached this far I think they're high enough intent that I'd think they're probably being a little more careful in what they're doing. I don't think diffs are a likely ball bounce for them to ... bounce along with.